cloudblue_connect.custom_adapter.rb

```diff
@@ -0,0 +1,694 @@
+          { name: "asset", type: "object", properties: [
+            { name: "id" },   
+            { name: "external_id" }
+          ]},
+          { name: "errors", type: "array"},
+        ]
+      end
+  
+    },    
+    purchase_request: {
+      fields: lambda do
+        [
+          { name: "id"},
+          { name: "type"},
+          { name: "status" },
+          { name: "params_form_url"},
+          { name: "asset", type: "object", properties: [
+            { name: "id" },   
+            { name: "external_id" }
+          ]}
+        ]
+      end
+  
+    },
+    change_request: {
+      fields: lambda do
+        [
+          { name: "id"},
+          { name: "type"},
+          { name: "error_code"},
+          { name: "status"},
+          { name: "asset", type: "object", properties: [
+            { name: "id"},
+            { name: "external_id"},
+            { name: "external_uid"}
+            ]
+          },
+          { name: "errors", type: "array"},
+          
+          
+         ]
+      end
+    },
+    request: {
+      fields: lambda do
+        [
+          { name: "id"},
+          { name: "type"},
+          { name: "note"},
+          { name: "reason"},
+          { name: "created"},
+          { name: "updated"},
+          { name: "answered""note"},
+          { name: "reason"},
+          { name: "assignee"},
+          { name: "activation_key"},
+          { name: "status" },
+          { name: "asset", type: "object", properties: [
+            { name: "id" },
+            { name: "status" },
+            { name: "external_id" },
+            { name: "external_uid" },
+            { name: "product", type: "object", properties: [
+              { name: "id" },
+              { name: "icon" },
+              { name: "name" },
+              { name: "status" }
+            ]},
+            # skipped connection and events structures
+            { name: "items", type: "array", properties: [
+              { name: "id" },
+              { name: "global_id" },
+              { name: "mpn" },
+              { name: "old_quantity" },
+              { name: "quantity" },
+              { name: "type" },
+              { name: "display_name" },
+              { name: "period" },
+              { name: "item_type" }
+            ]},
+            # skipped params array, tiers and configuration structures
+            { name: "marketplace", type: "object", properties: [
+              { name: "id" },
+              { name: "name" },
+              { name: "icon" }
+            ]}
+            # skipped contract structure
+          ]}
+        ]
+      end
+    },
+  },
+  methodes: {
+  },
+  actions: {
+
+
+    asset_cancel_request: {
+      title: "Create Asset Cancel Request",
+      subtitle: "",
+      description: "",
+      help: "<a href='https://connect.cloudblue.com/community/api/openapi/#operations-Assets-request_list_listRequests' target='_blank'>Official documentation</a>",
```

Comment:
this looks like incorrect URL. Correct URL is https://connect.cloudblue.com/community/api/openapi/#operations-Assets-request_list_createRequest